### DISCUSSION TITLE
Why aren't emojis in color ？

### DISCUSSION THREAD

**Discussion by ice909** [adityatelange/hugo-PaperMod#1301] [Q&A] [answered]
Why aren't emojis in color ？
This is my website
![image](https://github.com/adityatelange/hugo-PaperMod/assets/59855569/9dbd9dc9-ed77-46f6-888d-f22112dba40c)

This is a screenshot of the theme sample
![screenshot](https://github.com/adityatelange/hugo-PaperMod/assets/59855569/07d5c267-c361-461d-9d69-b945db293457)

**Comment by adityatelange** [comment]
https://stackoverflow.com/questions/66867069/emoji-display-black-and-white-windows7-chrome-edge#:~:text=Color%20fonts%20are%20not%20supported,consider%20upgrading%20to%20Windows%2010.

**Reply by ice909** [reply]
Sorry, I didn't mention in advance, but I'm using deepin

**Reply by adityatelange** [reply, marked as answer]
What I want to mention is it is subjective and depending upon the OS support to display color emojis, can't fix it in theme.
